### ISSUE TITLE
manufacturing for diamond

### ISSUE THREAD

**Issue by jacksolutions** [odoomrp/odoomrp-wip#1004]
i am not sure such a module is available that supports the diamond manufacturing process. manufacturing that comes with odoo doesnt handle this, i am not sure if any of the modules that is odoomrp supports that.

rough diamonds comes in a packet and lot and each diamond goes from a process rough to polished. 
so it starts manufacturing from rough to polished and during that process it goes to various sub processes. laser sawing, hand sawing, washing, polishing.

and on each stage "products" properties gets changed, i.E its weight, its size, its shape, its color, its height.. 

so how can we track on which manufacturing stage when it enters and when it leaves, what properties are changed, track of them is crucial in diamond manufacturing.. 

any modules that works like this process?

**Comment by anajuaristi** [comment]
Yes. We could offer to you one possible solution with modules built for odoomrp.
Would you be available to speak by skype or hangouts this week?
Than you!!

**Comment by jacksolutions** [comment]
okay so you mean to say you already have a modules in odoomrp which supports above functionalities? 

i have seen all the listed modules in odoomrp but none of them satisfy those funcationalities.! 
can you please list down those modules here which can do the needful here? so that we can discuss further?

**Comment by anajuaristi** [comment]
Yes. It's only the way you configure certain modules. Just a few clues:
You can manufacture by operations, so you have got your laser sawing, hand sawing, washing, polishing subprocesses covered. 

On the other side, you only need to register a set of data like  its weight, its size, its shape, its color, its height.. on each step... you can use quality modules + trazeability for that.

It's quite easy to cover your needs.

If you are interested we will start an english OdooMRP training first week october so you could learn how it works. If you would like to attend please, let me know on ajuaristio @ gmail.com
